source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,116 @@
+.. _csharp-time-series:
+
+=======================
+Time Series Collections
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use and interact with **time series collections** in
+MongoDB using the {+driver-long+}.
+
+Time series collections efficiently store sequences of measurements over
+a period of time. Time series data consists of any data collected over
+time, metadata that describes the measurement, and the time of the
+measurement.
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Example
+     - Measurement
+     - Metadata
+
+   * - Sales Data
+     - Revenue
+     - Company
+
+   * - Infection Rates
+     - Amount of People Infected
+     - Location
+
+Create a Time Series Collection
+-------------------------------
+
+.. important::
+
+   Time series collections require MongoDB 5.0 or later.
+
+To create a time series collection, pass the following parameters to the
+``CreateCollection()`` method:
+
+- Name of the new collection to create
+- A ``CreateCollectionOptions`` object that contains a ``TimeSeriesOptions`` object
+
+.. literalinclude:: /includes/fundamentals/code-examples/TimeSeries.cs
+   :start-after: begin-time-series
+   :end-before: end-time-series
+   :language: csharp
+   :dedent:
+
+To check if you successfully created the collection, send the
+``"listCollections"`` command to the ``RunCommand()`` method.
+
+.. literalinclude:: /includes/fundamentals/code-examples/TimeSeries.cs
+   :start-after: begin-run-command
+   :end-before: end-run-command
+   :language: csharp
+   :dedent:
+
+Your output will look similar to the following:
+
+.. code-block:: json
+
+    {
+    "cursor":{
+        ...
+        "firstBatch":[
+            {
+                "name":"september2021",
+                "type":"timeseries",
+                "options":{
+                "timeseries":{
+                    ...
+                }
+                },
+                "info":{
+                "readOnly":false
+                }
+            },
+    ...   
+    }
+
+Query a Time Series Collection
+------------------------------
+
+To query a time series collection, follow the conventions for 
+:doc:`retrieving </fundamentals/crud/read-operations/retrieve>` and aggregating data.
+
+Additional Information
+----------------------
+
+To learn more about the operations mentioned, see the following
+guides:
```

Comment:
S: I think it would be helpful to specify that these are Server manual guides
```suggestion
To learn more about the operations mentioned on this page, see the following
Server manual guides:
```